JS/Lesson24/goksel/script.js

```diff
@@ -0,0 +1,40 @@
+let formElement = document.getElementById("myForm");
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+formElement.addEventListener("submit", function (event) {
+
+    event.preventDefault();
+
+    let isFormCorrect = true;
+
+    // Variables for to check if email format is valid
+    let inputElement = document.getElementById("email");
+    let emailErrorElement = document.getElementById("email-error");
+
+    //Variables to change .active-form class after submitted
+    let formContainer = document.querySelector(".form-container");
+    let confirmContainer = document.querySelector(".confirm-container");
+
+    //Variable for to use user email on next form inside of the span content
+    let userEmail = document.getElementById("confirm-email");
+
+    if (!emailRegex.test(inputElement.value)) {
+        emailErrorElement.textContent = "Please enter a valid email";
+        isFormCorrect = false;
+        inputElement.style.color = "#FF6253";
```

Comment:
Don't  use inline styles, instead, add a class (e.g. 'error') here and modify styles in CSS file to change color when the class is applied.